confidential/confidential.go

```diff
@@ -125,3 +125,65 @@ func ValueCommitment(value uint64, generator []byte, factor []byte) (result [33]
 	result = commit.Bytes()
 	return
 }
+
+type SurjectionProofInput struct {
+	OutputAsset               []byte
+	OutputAssetBlindingFactor []byte
+	InputAssets               [][]byte
+	InputAssetBlindingFactors [][]byte
+	Seed                      []byte
+}
+
+//SurjectionProof method generates surjection proof
+func SurjectionProof(input SurjectionProofInput) ([]byte, error) {
+	ctx, _ := secp256k1.ContextCreate(secp256k1.ContextBoth)
+	defer secp256k1.ContextDestroy(ctx)
+
+	outputGenerator, err := secp256k1.GeneratorGenerateBlinded(ctx, input.OutputAsset, input.OutputAssetBlindingFactor)
```

Comment:
Take a look at [this comment](https://github.com/vulpemventures/go-elements/pull/72/files#r434452789) about code style guides.

Reply:
Done